en/deploy-tidb-binlog.md

```diff
@@ -124,35 +131,16 @@ TiDB Binlog is disabled in the TiDB cluster by default. To create a TiDB cluster
                       - key: "app.kubernetes.io/instance"
                         operator: In
                         values:
-                        - <release-name>
+                        - <cluster-name>
                     topologyKey: kubernetes.io/hostname
         ```
+    > **Note:**
+    >
+    > If you update the affinity configuration of the TiDB components, it causes rolling updates of the TiDB components in the cluster.
```

Comment:
```suggestion
    > If you update the affinity configuration of the TiDB components, it will cause rolling updates of the TiDB components in the cluster.
```